screens/ChefMealScreen.js

```diff
@@ -0,0 +1,18 @@
+/******************************************
+ *  Author : Gil
+ *  Created On : Fri Apr 10 2020
+ *******************************************/
+
+import React from "react";
+import { MEALS } from "../data/dummy-data";
+import MealsList from "../components/MealsList";
+
+const ChefMealScreen = (props) => {
+  let { chefName } = props;
+  chefName = chefName ? chefName : "גיל רותם";
```

Comment:
Why is גיל the default?
There shouldn't be a default. We have to pass the chef's name.